elasticsearch/jvmopts.sls

```diff
@@ -0,0 +1,18 @@
+include:
+  - elasticsearch.service
+
+{%- set major_version = salt['pillar.get']('elasticsearch:major_version', 2) %}
+
+{%- if major_version >= 5 %}
+{%- set jvm_opts = salt['pillar.get']('elasticsearch:jvm_opts') %}
+{%- if jvm_opts %}
```

Comment:
> By not combining conditionals there's a performance boost on older versions since you don't do the pillar lookup. That's why jvm_opts is set inside the conditional.

How did you measure this performance boost?

> Also, adding a default value for jvm_opts will cause the formula to overwrite the default jvm.options file, which I'd rather avoid. I'd prefer this to be an optional thing... if the user sets the values in Pillar then we'll overwrite the default file.

Using `jvm_opts = salt['pillar.get']('elasticsearch:jvm_opts', '{}')` will not render the `jvm.options` file. Therefore, using a default for `jvm_opts` will not overwrite the original.